ig/README.md

```diff
@@ -32,3 +33,24 @@ A Dockerfile is provided that can be used to generate the Implementation Guide.
 1. The container also pre-generates the documentation.  This process could make the image build take 10-15 minutes
 1. All of the generated documentation should be in the `/trusted-intermediary/output` directory
 1. If building locally, you may want to add a `--no-cache` to the `docker buildx build` command so that the documentation generation is not cached
+
+## Using Docker for local development
+
+Provided is a Dockerfile suitable for running the implementation guide documentation generation without the need to install all the tooling locally.
+The Docker image needs to be built and then run with an appropriate `mount` flag in order to mount your local `ig` directory into the running Docker container.
+This will allow bi-directional writes so that the Docker container can generate the documentation, and as a developer you can view the generated documentation easily.
+A developer will also be able to update any input files for the documentation and re-run the documentation generation without any intermediate steps.
+
+1. In a terminal, navigate to this `ig` directory
+2. run `docker buildx build -t ig-local-dev -f Dockerfile-local-dev .` to build the Docker container
+3. run `docker run -it --mount type=bind,source="$(pwd)",target=/trusted-intermediary ig-local-dev bash` to run the container and use an interactive `bash` shell inside the container
```

Comment:
Will the shorthand form work too?

`docker run -it -v "$(pwd)":/trusted-intermediary ig-local-dev bash`

Reply:
According to the documentation in order to do a two way bind type of volume mount you need to use the `--mount` flag and not the `-v` flag.

Reply:
I did a quick test, and `-v` seems to reflect any changes in the Docker container's filesystem in the hosts' file system.  I looked at some [documentation](https://docs.docker.com/storage/bind-mounts/), and Docker recommends using the new `--mount` argument, so I'm fine with whatever.